source/target/nordic/nrf5x/target_16.c

```diff
@@ -0,0 +1,37 @@
+/**
```

Comment:
Will remove it from the nRF5x folder. Since other projects are using the nrf51 folder, i will not remove or alter the nrf51 folder.